### DISCUSSION TITLE
First-party cookies

### DISCUSSION THREAD

**Discussion by helenafranca** [GoogleChromeLabs/ps-analysis-tool#629] [Support Forum] [answered]
Hi! I believe the answer is yes, but can I assume that a **first-party cookie** shown in PSAT will not break even when PSAT shows a third-party platform related to this same cookie?

![Screenshot 2024-04-17 10 54 20 AM](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/85519105/54769b6d-5efc-477f-a2c8-e932da980520)

**Comment by amedina** [comment, marked as answer]
Hi @helenafranca, that is correct. Although a cookie can be associated in some way with a 3P platform, it can be "configured" via the domain attribute as a 1P cookie. Those cookies are 1P for the current browsing context (`clarin.com` in your example), and therefore will not be affected by the deprecation of unrestricted 3P cookies.
